e2e/framework/util.go

```diff
@@ -0,0 +1,20 @@
+package framework
+
+import "time"
+
+func Retry(attempts int, delay time.Duration, f func() error) error {
```

Comment:
There are some good helpers in apimachinery for these kinds of things: https://github.com/kubernetes/apimachinery/blob/master/pkg/util/wait/wait.go#L184